tools/bin/ariaDownload.py

```diff
@@ -30,6 +30,8 @@ def createParser():
     parser.add_argument('-w', '--workdir', dest='wd', default='./products', type=str, help='Specify directory to deposit all outputs. Default is "products" in local directory where script is launched.')
     parser.add_argument('-s', '--start', dest='start', default=None, type=str, help='Start date as YYYYMMDD; If none provided, starts at beginning of Sentinel record (2014).')
     parser.add_argument('-e', '--end', dest='end', default=None, type=str, help='End date as YYYYMMDD. If none provided, ends today.')
+    parser.add_argument('-u', '--user', dest='user', default=None, type=str, help='NASA Earthdata URS user login. "ARIA Product Search" must be a URS approved application.')
+    parser.add_argument('-p', '--pass', dest='passw', default=None, type=str, help='NASA Earthdata URS user password. "ARIA Product Search" must be a URS approved application.')
```

Comment:
@bbuzz31 this is not fully correct. The ARIA product search is for the aria-products page so not needed for downloading using the DAAC api. What is needed are:

- GRFN Door (PROD)
- ASF Datapool products